centreon/src/Core/Infrastructure/Configuration/MetaService/Repository/DbReadMetaServiceRepository.php

```diff
@@ -38,6 +40,49 @@ public function __construct(DatabaseConnection $db)
         $this->db = $db;
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function findNames(array $metaServiceIds): MetaServiceNamesById
+    {
+        $bindValues = [];
+
+        foreach ($metaServiceIds as $index => $metaId) {
+            $bindValues[":meta_{$index}"] = $metaId;
+        }
```

Comment:
you could use the same pattern as in DbReadContactGroupRepository.php with the string concatenation